wpxmlrpc.podspec

```diff
@@ -1,16 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "wpxmlrpc"
-  s.version      = "0.8"
+  s.version      = "0.8.2"
   s.summary      = "Lightweight XML-RPC library."
   s.homepage     = "https://github.com/wordpress-mobile/wpxmlrpc"
-  s.license      = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.author       = "WordPress"
+  s.license      = { :type => 'GPLv2', :file => 'LICENSE.md' }
+  s.author       = {"WordPress", "Automattic" => "mobile@automattic.com"}
```

Comment:
I was under the assumption that all wordpress related projects needed to be under GPLv2. But if I'm wrong I can revert this.